Evaluation/Input/evaluation_plan.json

```diff
@@ -123,24 +123,27 @@
             {
                 "Project": "Compound",
                 "Simulation": "Sim 1 - intravenous",
-                "SectionId": 33
+                "SectionReference": "ct-profiles"
             },
             {
                 "Project": "Compound",
                 "Simulation": "Sim 2 - peroral",
-                "SectionId": 33
+                "SectionReference": "ct-profiles"
             }
         ],
         "GOFMergedPlots": [
             {
                 "Title": "Goodness of fit plor for concentration in plasma",
-                "SectionId": 32,
+                "SectionReference": "diagnostics-plots",
                 "Artifacts": [
                     "Plot",
                     "GMFE",
                     "Measure"
                 ],
-                "PlotType": "predictedVsObserved|residualsOverTime",
+                "PlotTypes": [
+                    "predictedVsObserved",
+                    "residualsOverTime"
+                ],
```

Comment:
another change: use **PlotTypes** as array instead of concatenated string. 
Same as with the SectionId: the old "PlotType" keyword is still supported at the moment, but not forever